src/blueprints/tv2/timeline-object-factories/tv2-caspar-cg-timeline-object-factory.ts

```diff
@@ -65,18 +228,20 @@ export class Tv2CasparCgTimelineObjectFactory {
   }
 
   public createDveLocatorTimelineObject(): CasparCgTemplateTimelineObject {
-    return {
-      id: 'casparCg_locators',
-      enable: {
-        start: 0
-      },
-      priority: 1,
-      layer: Tv2GraphicsLayer.GRAPHICS_LOCATORS,
-      content: {
-        deviceType: DeviceType.CASPAR_CG,
-        type: CasparCgType.TEMPLATE
-      }
-    }
+    return {} as CasparCgTemplateTimelineObject
+    // TODO: RKLI Should implement this
+    // return {
+    //   id: 'casparCg_locators',
+    //   enable: {
+    //     start: 0
+    //   },
+    //   priority: 1,
+    //   layer: Tv2GraphicsLayer.GRAPHICS_LOCATORS,
+    //   content: {
+    //     deviceType: DeviceType.CASPAR_CG,
+    //     type: CasparCgType.TEMPLATE
+    //   }
+    // }
```

Comment:
Lindved commented it out and added the return statement on line 231, as it was not compiling, but the method in question is not one i am suppose it implement. It is a remaining method for DVE one CasparCG.

Reply:
It should have said that it should have been made in SOF-1695.

FYI, this changes does break the calculation of the Timeline since now we have a TimelineObject without a `content` which is not valid. It only happens if you do a SPLIT_SCREEN Action and I'm handling it in SOF-1696